packages/angular/projects/ui-angular/src/lib/primitives/amplify-password-field/amplify-password-field.component.ts

```diff
@@ -8,7 +9,7 @@ import { translate } from '@aws-amplify/ui';
 export class AmplifyPasswordFieldComponent {
   @Input() autocomplete = 'new-password';
   @Input() disabled = false;
-  @Input() id: string;
+  @Input() fieldId: string = `amplify-field-${nanoid(12)}`;
```

Comment:
Renamed this prop `id` to `fieldId` to prevent duplicate `id` html attribute values on screen, which breaks accessibility.

Reply:
Related, https://github.com/angular/angular/issues/18877